package.json

```diff
@@ -39,7 +39,12 @@
     "spectron": "jest \"tests/specs/.*\\.spec\\.js\" --maxWorkers=4",
     "spectron-inband": "jest \"tests/specs/.*\\.spec\\.js\" --runInBand",
     "spectron-coverage": "nyc report --reporter=text-lcov --temp-dir=tests/coverage | codecov --pipe --disable=gcov",
-    "generate-screenshots": "PWD=$(pwd); cd docker; docker run -v \"$PWD:/github/workspace\" -w \"/github/workspace\" --rm -it $(docker build -q .)"
+    "generate-screenshots": "PWD=$(pwd); cd docker; docker run -v \"$PWD:/github/workspace\" -w \"/github/workspace\" --rm -it $(docker build -q .)",
+    "pretest": "tsc --incremental -p playwright/tsconfig.json",
+    "playwright": "cross-env ALLURE_RESULTS_DIR=playwright/artifacts/allure-results npx playwright test --config=playwright/playwright.config.ts",
+    "playwright-ci": "nyc yarn playwright",
```

Comment:
Wonder if it might be better to rename this `playwright-code-coverage` or something to make it clearer what it's doing. Unless it's also doing something specific to the CI as well?